addons/account/views/account_journal_dashboard_view.xml

```diff
@@ -30,7 +30,7 @@
                     </t>
 
                     <t t-name="JournalTop">
-                        <div class="o_kanban_card_header">
+                        <div t-attf-class="o_kanban_card_header #{kanban_color(1)}">
```

Comment:
Because before this changes we were applying color on whole kanban and at that time we need kanban box white by default.  That's why we wrote color(0). In account journal module there is no color field in customization of kanban, so by default header of kanban should be grey for that particular module. Header can not be white by default because it looks weird. So, i changed it from color(0) to color(1)... thanx...